analyzers/tests/SonarAnalyzer.UnitTest/Extensions/ITupleOperationWrapperExtensionsTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.CodeAnalysis.CSharp;
+using SonarAnalyzer.Extensions;
+using StyleCop.Analyzers.Lightup;
+
+namespace SonarAnalyzer.UnitTest.Extensions
+{
+    [TestClass]
+    public class ITupleOperationWrapperExtensionsTests
+    {
+        [DataTestMethod]
+        [DataRow("_ = (1, 2);", "1", "2")]
+        [DataRow("_ = (x: 1, y: 2);", "1", "2")]
+        [DataRow("_ = (1, M());", "1", "M()")]
+        [DataRow("_ = (1, (2, 3));", "1", "2", "3")]
+        [DataRow("_ = (1, (2, 3), 4, (5, (6, 7)));", "1", "2", "3", "4", "5", "6", "7")]
+
```

Comment:
Empty lines should not be between data rows. Either place a comment there describing what's special about the next block or remove it please.